package.json

```diff
@@ -7,52 +7,54 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "NODE_TLS_REJECT_UNAUTHORIZED=0 gatsby develop --host 0.0.0.0 --https",
+    "develop:vanilla": "gatsby clean && gatsby develop --host 0.0.0.0",
     "develop:expose": "gatsby clean && npm run develop",
     "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
     "start": "npm run develop",
-    "serve": "gatsby serve",
+    "serve": "gatsby serve -p 5000",
```

Comment:
Is this necessary?

Reply:
In case you want to run this side to side to the `develop` script then yes. Albeit, that would hardly happen

Reply:
Ah I see, wouldn't it automatically pick the next open port without a specified `-p` parameter?

Reply:
You would think so, but it seems more rigid than we thought. 

Though I'll remove it. It can also be done in the cli `npm run serve -- -p 5000`